packages/driver/src/cypress.js

```diff
@@ -167,6 +167,23 @@ class $Cypress {
     return this.runner.run(fn)
   }
 
+  // Callback to execute when module.hot is called in component testing
+  restartRunner () {
+    if (!window.parent.Cypress || !window.parent.Cypress.$autIframe) {
+      throw Error('Cannot re-run spec without Cypress or Cypress.$autIframe')
+    }
+
+    const $autIframe = window.parent.Cypress.$autIframe[0]
+    // MobX state is only available on the Runner instance
+    // which is attached to the top level `window`
+    const state = $autIframe.ownerDocument.defaultView.Runner.state
```

Comment:
We can just go into window.parent.Runner, right?

Reply:
Surprisingly no, it doesn't exist there

Reply:
whut